src/lib/commandBarConfigs/projectsCommandConfig.ts

```diff
@@ -26,147 +18,240 @@ export type ProjectsCommandSchema = {
   }
 }
 
-export const projectsCommandBarConfig: StateMachineCommandSetConfig<
-  typeof projectsMachine,
-  ProjectsCommandSchema
-> = {
-  'Open project': {
-    icon: 'arrowRight',
-    description: 'Open a project',
-    status: isDesktop() ? 'active' : 'inactive',
-    args: {
-      name: {
-        inputType: 'options',
-        required: true,
-        options: (_, context) =>
-          context?.projects.map((p) => ({
-            name: p.name,
-            value: p.name,
-          })) || [],
+export const openProjectCommand: Command = {
+  icon: 'arrowRight',
+  name: 'Open project',
+  displayName: `Open project`,
+  description: 'Open a project',
+  groupId: 'projects',
+  needsReview: false,
+  onSubmit: (record) => {
+    if (record) {
+      systemIOActor.send({
+        type: SystemIOMachineEvents.navigateToProject,
+        data: { requestedProjectName: record.name },
+      })
+    }
+  },
+  args: {
+    name: {
+      required: true,
+      inputType: 'options',
+      options: () => {
+        const folders = folderSnapshot()
+        const options: CommandArgumentOption<string>[] = []
```

Comment:
Pardon me for pushing to your branch but asking for this tiny type clarification to remove `any` didn't seem worth your time

Reply:
:cheese: